[V] Sorokyne Strata: Expanded Small corridors, removed 3000 unused tiles, Xenos can now weed everything except LZ's and surrounding of LZ's, allow LAZ for CAS inside structures with light blue Areas on Tacmap (glass ceiling hallways)

Venuska1117 opened the pull request:
# About the pull request

This PR clean Sorokyne Strata file from excessive lines of code caused by APC's and small light fixture.
It changes some small rooms with have 1x1 hallways, now if possible, they are 2x2 (minimum).
Replaced 80% of indestructable walls with heavy reinforced walls, i dont see need for indestructable walls if they are not critical (for example, map edge)
Shortened river by half (south west near LZ) and placed invisible wall, i tough of preventing (in future) for burrowers to getting there and re-evolve to queen in spot where marines have no access because of invisible wall.
It "removes" walls with are 2tiles or more thick, i don't know why there are 2 tiles thick walls, but i believe they are not important.
Removes around 3000 unused tiles.



Edit: 08.08
- Allow xenos to weed everywhere (except LZ's)
- Allow marines to use LAZ for CAS inside most structures that have light blue colored areas (hallways made of glass ceiling)

Fixes: #6869

# Explain why it's good for the game

Less lines of code allow for quicker groundsite loading, even if its second faster, and also allow for more map destruction with less chockes caused by 1x1 tunnels, they don't appear much but they are "annoying", allow marines to use LAZ for CAS inside most structures (not all of them) and xenos can now weed 90% of areas except LZ's

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

![img1](https://github.com/cmss13-devs/cmss13/assets/89580971/dd995604-8141-40eb-b6a2-702868667bd4)

![img2](https://github.com/cmss13-devs/cmss13/assets/89580971/050f13ee-057d-4d6f-bdbd-8dccbf24aadd)

![img3](https://github.com/cmss13-devs/cmss13/assets/89580971/50fc2692-3429-414b-bd3b-37fca09030eb)

![img4](https://github.com/cmss13-devs/cmss13/assets/89580971/00265e18-b9a7-444e-8ae4-4ed0e3812ad1)

![img5](https://github.com/cmss13-devs/cmss13/assets/89580971/f064ad40-f6ac-46ec-bee1-fdc3d0bab54b)

![img6](https://github.com/cmss13-devs/cmss13/assets/89580971/bda1b51a-84e8-4397-89cf-badf81cfe9a0)

![img7](https://github.com/cmss13-devs/cmss13/assets/89580971/245ee13c-48a0-4630-aee7-fa7ce9172b5d)

![img8](https://github.com/cmss13-devs/cmss13/assets/89580971/f0847b36-6c7d-486a-a1e7-9cf82dcc2daa)

![img9](https://github.com/cmss13-devs/cmss13/assets/89580971/179f0fe4-31c0-47a5-b0dc-24f058d9bd40)


</details>


# Changelog

:cl: Venuska1117
maptweak: Sorokyne Strata - made most indestructable walls destructable.
maptweak: Sorokyne Strata - changed rooms/hallways with have 1x1 hallways to be wider.
maptweak: Sorokyne Strata - replaced some small light fixtures with normal light fixtures.
maptweak: Sorokyne Strata - reduced 2 tiles or more thick walls to 1 tile thick walls.
balance: Made most areas weedable, except LZ's zones and surrounding LZ's sub-variants.
balance: Allow LAZ inside Outposts (mining and normal ones) as long as they aim at light blue areas (glass ceiling hallways).
code: Changed pixel offset for "tube" lights to be closer to "floor".
/:cl:

Nanu308 commented on maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/c216e0cf-e950-4209-90be-73fa33d5d037)
Check if you can see down here with binocs/scopes, if so, add in proper walls if not it risks breaking immersion.

![image](https://github.com/cmss13-devs/cmss13/assets/59782240/e1f7b7cb-35b9-4b73-a596-d36889dd5743)
Door direction is wrong.


Other than that looking good 👍

Venuska1117 commented:
@Nanu308 can you please check map again, i removed changed south part of south jungle to remove around 3000-4000 unused tiles (considering i already removed long river) and moved some queen "nest" spots to better points (like one between hot springs and carp lake) more south of hot springs, i did this changes (movin nests) because by gameplay perspective you are unable to weed  water/coast.

Nanu308 commented on maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/a65f01f7-c862-4ff9-933a-3b139e7328c3)
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/32f8479a-596d-40ea-a104-65558eefea44)
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/2cc547e7-8cc7-4351-ad97-2ee6edfcfd9f)
Areas and stone walls, keep consistent

![image](https://github.com/cmss13-devs/cmss13/assets/59782240/2c4071f4-b2a6-473b-8c48-4e257108fa1e)
The room ends at the flooring, area here as well

![image](https://github.com/cmss13-devs/cmss13/assets/59782240/6927fea2-4359-4d83-b207-e55291586828)
Areas and floor tiles consistency. 

![image](https://github.com/cmss13-devs/cmss13/assets/59782240/bcd4153a-ac73-4f12-8deb-37654325d5e2)
Door needs to be indestructible. 

![image](https://github.com/cmss13-devs/cmss13/assets/59782240/b9dc5606-6c73-4a68-b8ac-540d287f0848)
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/eeeb60fb-5a48-4c6d-a9f4-b88c524c6f1f)
Needs a light or two extra.


Go over the areas and the lighting, but good changes overall to the layout which should help a fair bit with the claustrophobic feeling of the map for the marines in particular.

Venuska1117 replied:
most of this area stuff was there by default, i didnt made those changes but thanks for checking them up, i will change them in next pull

Nanu308 commented on maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:
Need light/s:
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/0bf4b5a8-f5ca-4249-836c-84b798c1aab0)
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/b1746280-47f1-45a4-a91c-435236094d60)
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/a874612a-09c3-4f63-a966-e7759d3c1c09)
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/e76a4d17-779a-42e0-b8a8-7ff47642f15a)


Semiotic symbols, Check over computers, APC's, Restricted Areas such as armories, security for security areas, and so forth. Check the pinned message in the mapper channel for the general guide. For examples look at Almayer, it should be very up to par currently.

Some examples:
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/1f62e631-22cf-4a4d-bb3d-27a63811919c)
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/2525b308-1889-4b8c-9941-8b9df3c165e2)
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/27ed204d-18db-47a5-9064-accfed03f59b)
![image](https://github.com/cmss13-devs/cmss13/assets/59782240/e608a606-3007-478c-9a47-52e3d9b64ccf)


PR coming along nicely, put it in for testing 👍

Venuska1117 replied:
most of small light got removed and replaced with "long" lights, and to be honest, blub lights on glass windows looked strange (and they can be destroyed so sometimes it look like they float)

Drulikar commented:
Stale exempt until Nanu makes a decision on this.

Drulikar commented:
Be sure to replicate the change in https://github.com/cmss13-devs/cmss13/pull/6494 that you will likely conflict with.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

Drulikar reviewed (changes requested):
Its long overdue, but for https://github.com/cmss13-devs/cmss13/pull/6363 can you please expand the LZ or manipulate the areas (do so in code edits not var edits ideally) so that the LZ gas is how it would be done on other maps?

The gas is determined by whether the `is_landing_zone` var is set on the area. But note this also determines whether the area is mortar-able if lz protections are on. It also should not be too far from the dropship console because it is the dropship console that makes the fake OB to warn survivors they need to get out of there immediately.

You can test this in localhost by temporarily editing the `LZ_HAZARD_START` var in colonialmarines.dm and using the game panel (after load) to change the game mode to distress signal.

At the moment the gas looks like this on your version of the map (with my suggestions):
![image](https://github.com/user-attachments/assets/03bbf4a2-3a4a-4bae-bd72-3c692fdf2690)
![image](https://github.com/user-attachments/assets/702541e8-08ca-4341-92e4-1af208860c61)

Drulikar commented on maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:
Needs to be expanded here (this area is reachable):
![image](https://github.com/user-attachments/assets/ce121968-51e0-44fc-a8f8-81628114f089)

Consider expanding it here some:
![image](https://github.com/user-attachments/assets/44665ac5-5141-44f9-9551-c846a181576a)

Consider shrinking it here some:
![image](https://github.com/user-attachments/assets/e6d52545-68ab-45b4-a093-d1ae2f79924f)

Drulikar commented:
Gas looks fine now

Git-Nivrak commented:
Most likely the cause for #6869

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

Nanu308 commented on maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:
![image](https://github.com/user-attachments/assets/59211044-0884-4242-8572-b135ed778aac)
![image](https://github.com/user-attachments/assets/2a2d2893-61bd-4e5c-9105-f0a93ad90bfa)
![image](https://github.com/user-attachments/assets/d11e62f3-0e3c-4eab-81ec-f3cb764d52db)
![image](https://github.com/user-attachments/assets/e9c7e447-b023-4dc0-a385-1cc8c7af1396)
![image](https://github.com/user-attachments/assets/af445483-103f-49bf-9025-d0f5558ff49c)
![image](https://github.com/user-attachments/assets/918f0c8f-05e7-4bf4-aca2-470debf98b24)
Pipes below walls/windows. Double check the system and route through areas that are open or under doors.

Cant spot anything else. Overall good improvement to the map. Can be merged after these fixes, might need further tweaks down the line.

Venuska1117 replied:
Changed pipes to follow standards: 
1. Dont go trough undestructable wall
2. Prioritize doors, if there is no access then window frames.
3. Make pipes as straight in line as possible, attach vents using closest path possible. 

(thats general rules I follow and applied it to your request, to make it look more logical.)

Nanu308 commented on maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:
Might be some oversights,  but nothing immediately popping out.

Venuska1117 replied:
Great! Thank you!
